scheduler/crates/api/src/lib.rs

```diff
@@ -96,8 +96,10 @@ impl Application {
             .await
             .is_none()
         {
-            let mut account = Account::default();
-            account.secret_api_key = secret_api_key;
+            let mut account = Account {
+                secret_api_key,
+                ..Default::default()
+            };
             if let Ok(mut verification_key) = std::env::var("ACCOUNT_PUB_KEY") {
                 verification_key = verification_key.replacen("\\n", "\n", 100);
```

Comment:
@fmeringdal Why do we need this line?

Reply:
Are you thinking of `secret_api_key` at line 100? This is so that the default `Account` is using the `secret_api_key` that is set by the environment variable.

Reply:
No, I'm asking about `verification_key = verification_key.replacen("\\n", "\n", 100);`

Reply:
It was kind of a hassle to parse rsa keys (public rsa keys are used for verifying jwt tokens) from environment variables because of newlines.
[This stackoverflow thread](https://stackoverflow.com/questions/43082918/how-to-set-multiline-rsa-private-key-environment-variable-for-aws-elastic-beans) explains the issue I had. 
Also I dont think rust has a `replace_all` method on strings, so I had to  use `replacen` with a numer like 100 which is just a number that is bigger than the number of lines of any public rsa keys

Reply:
OK, now I understood. Thanks.
Anyway, normal `replace` is `replace_all` by nature. (why anybody on earth needs a one-time replacer? and if they do, then it's replacen(1) :D )

https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=c5de093ce3d6b7004db539495fcf1c76

Reply:
You are correct, I dont know why I didnt think that `replace`  would replace all when I wrote that code, haha.
Would you like to submit a fix for this?

Reply:
You can push to master faster :D If you don't mind.